src/components/login/loginForm.tsx

```diff
@@ -1,73 +1,125 @@
-import './login.css';
 import { signInWithEmailAndPassword } from 'firebase/auth';
-import { useState } from 'react';
 import { auth } from '../../firebase-config';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
 
-function Login() {
-    const [email, setEmail] = useState('');
-    const [password, setPassword] = useState('');
+function Copyright(props: any) {
```

Comment:
Copyright is found in both the loginForm.tsx and signUp.tsx. That's a violation of the DRY principal (Don't Repeat Yourself). Please pull that function into it's own functional component and import it where ever it's needed.

Reply:
Done

Reply:
I had to add it back to the Login page as it is now showing separately from the sign up page